src/pages/PrintOnlyScreen.tsx

```diff
@@ -58,29 +67,28 @@ const PrintOnlyScreen = ({
   }, [markVoterCardPrinted, printer, updateTally])
 
   useEffect(() => {
-    if (!isEmptyObject(cardVotes)) {
+    if (isReadyToPrint && okToPrint) {
+      setOkToPrint(false)
+
       printBallot()
     }
-  }, [cardVotes, printBallot])
+  }, [cardVotes, printBallot, isReadyToPrint, okToPrint, setOkToPrint])
 
   useEffect(() => {
     if (!isVoterCardPresent) {
       updateIsPrinted(false)
+
+      // once card is taken out, ok to print again
+      if (!okToPrint) {
```

Comment:
Is this conditional check is unnecessary. Setting state to what it already is does not cause a re-render.